src/controllers/user/user-pool.js

```diff
@@ -0,0 +1,9 @@
+
+import { CognitoUserPool } from 'amazon-cognito-identity-js'
+
+const poolData = {
+  UserPoolId: 'us-east-1_OkSTj3mq4',
```

Comment:
Let's parameterize these variables (use these values as defaults if you want).

Reply:
Done